docs/data/data-grid/tutorials/server-side-data/index.md

```diff
@@ -0,0 +1,1054 @@
+---
+title: Tutorial - Data Grid with Server-side Data
+description: Learn how to use the Data Source layer to streamline communications between the Data Grid and server-side data.
+---
+
+# Build a Data Grid with Server-side Data
+
+## Overview
+
+This tutorial walks you through building a Data Grid that fetches data from a server with pagination, sorting, and filtering.
+The primary purpose is to demonstrate the use of the [Data Source layer](/x/react-data-grid/server-side-data/), which streamlines the development of key Data Grid features when working with server-side data by providing an interface for communications between the Grid on the client and the data on the server.
+
+## Prerequisites
+
+- Basic React knowledge
+- Understanding of TypeScript interfaces
+- Familiarity with async/await and fetch API
+
+### Relevant documentation
+
+The docs listed below may be useful if you're new to the MUI X Data Grid:
+
+- [Server-side data with the Data Source layer](/x/react-data-grid/server-side-data/)
+- [Column definition](/x/react-data-grid/column-definition/)
+- [Pagination](/x/react-data-grid/pagination/)
+- [Sorting](/x/react-data-grid/sorting/)
+- [Server-side filtering](/x/react-data-grid/filtering/server-side/)
```

Comment:
Linking this might create some confusion as the data source layer doesn't use this method for server-side filtering.
```suggestion
```

Reply:
Ah good catch! Should I link to the main filtering page or just leave it out?